src/interfaces/IServiceManagerUI.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.5.0;
+
+import {ISignatureUtils} from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
+import {IDelegationManager} from "eigenlayer-contracts/src/contracts/interfaces/IDelegationManager.sol";
+
+/**
+ * @title Minimal interface for a ServiceManager-type contract that AVS ServiceManager contracts must implement
+ * to index their data for the V1 AVS Marketplace
```

Comment:
can we change this to: "[...] must implement for eigenlabs to be able to index their data [...]"? Otherwise sounds like this indexing is something they would want to do themselves and the AVS teams might get confused.

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/242/commits/cbe2107dd590fd49c5aa97274ca7beecb9b19644
Changed to 
```
 * @title Minimal interface for a ServiceManager-type contract that AVS ServiceManager contracts must implement
 * for eigenlabs to be able to index their data on the AVS marketplace frontend.
```